README.rst

```diff
@@ -22,15 +22,32 @@ respy
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
     :target: https://github.com/psf/black
 
+----
 
-``respy``  is an open-source Python package for the simulation and estimation of a
-prototypical finite-horizon discrete choice dynamic programming model. We build on the
-baseline model presented in:
+``respy`` is an open source framework written in Python for the simulation and
+estimation of some finite-horizon discrete choice dynamic programming models. The group
+of models which can be currently represented in ``respy`` are called
+Eckstein-Keane-Wolpin models (Aguirregabiria and Mira (2010))
 
-    Keane, M. P. and  Wolpin, K. I. (1994). `The Solution and Estimation of Discrete
-    Choice Dynamic Programming Models by Simulation and Interpolation: Monte Carlo
-    Evidence <https://doi.org/10.2307/2109768>`_. *The Review of Economics and
-    Statistics*, 76(4): 648-672.
+What makes ``respy`` powerful is that it allows to build and solve structural models in
+minutes whose development previously took years. The design of ``respy`` allows the
+researcher to flexibly add the following components to her model.
+
+- **Any number of two or more choices** (e.g., working alternatives, schooling, home
+  production, retirement) where each choice may yield a wage, may allow for experience
+  accumulation and can be constrained by time, a maximum amount of accumulated
+  experience or other characteristics.
+
+- Condition the decision of individuals on its **previous choices** or their labor
+  market history.
+
+- Adding a **finite mixture** with any number of subgroups to account for unobserved
+  heterogeneity among individuals as developed by Keane and Wolpin (1997).
+
+- **Any number of additional characteristics** (e.g., ability measures (Bhuller et al.
+  (2020)), race (Keane and Wolpin (2000)), demographic variables) found in the data.
+
+- Estimate **measurement errors** in wages using a Kalman Filter.
```

Comment:
"correct the estimation for measurement error in wages, either using a Kalman filter in maximum likelihood estimation or by adding the measurement error in simulation based approaches"